utils/config/examples/daos_server_psm2.yml

```diff
@@ -85,3 +82,27 @@ servers:
   # bdev_list: [/tmp/daos-bdev]	# generate nvme.conf as follows:
               # [AIO]
               #   AIO /tmp/aiofile AIO1 4096
+-
+  targets: 8                # count of storage targets per each server
+  first_core: 25            # offset of the first core for service xstreams
+  nr_xs_helpers: 0          # count of offload/helper xstreams per target
+  fabric_iface: ib0         # map to OFI_INTERFACE=ib0
```

Comment:
This should be ib1, right?

Reply:
yes and the port number needs to be changed